### PR TITLE
Add products to line items on cart refresh

### PR DESCRIPTION
**Pull request by vkrashenko**

### CODE REVIEW

File: VirtoCommerce.Storefront/Domain/Cart/CartBuilder.cs [hunk 1]
```diff
@@ -82,12 +82,15 @@ public void LoadOrCreateNewTransientCart(string cartName, Store store, User user
                 //Load cart payment plan with have same id
                 if (store.SubscriptionEnabled)
                {
+                   var productsIds = cart.Items.Select(x => x.ProductId).Distinct().ToArray();
+                   var products = await _catalogService.GetProductsAsync(productsIds, ItemResponseGroup.ItemWithPrices);
```
Comment (tatarincev) [inline VirtoCommerce.Storefront/Domain/Cart/CartBuilder.cs]: Need to load products with these response groups 
ItemResponseGroup.ItemWithPrices | ItemResponseGroup.ItemWithDiscounts | ItemResponseGroup.Inventory

Comment (vkrashenko) [reply VirtoCommerce.Storefront/Domain/Cart/CartBuilder.cs]: cartbuildertest

**Comment by vkrashenko** [comment]
cartbuildertest

**Comment by vkrashenko** [comment]
**"The product for LineItem added to the cart was removed"**

_On this case we have message that product is unavailable_

image

**Comment by tatarincev** [comment]
released
https://github.com/VirtoCommerce/vc-storefront-core/releases/tag/v3.0.5

**Pull request merged**